benches/helpers.rs

```diff
@@ -150,8 +151,8 @@ pub async fn ws_server(handle: tokio::runtime::Handle) -> (String, jsonrpsee::se
 			UNSUB_METHOD_NAME,
 			|_params, pending, _ctx, _| async move {
 				let sink = pending.accept().await?;
-				let msg = SubscriptionMessage::from("Hello");
-				sink.send(msg).await?;
+				let json = RawValue::from_string("\"Hello\")".to_string()).unwrap();
```

Comment:
Nit: I wonder whether something like this API would be nicer:

https://docs.rs/serde_json/latest/serde_json/value/fn.to_raw_value.html

So eg `serde_json::value::to_raw_value("hello").unwrap()`